docs/UserGuide.adoc

```diff
@@ -37,6 +37,7 @@ What's different from AddressBook-Level1:
 * Support for storing address (`a/`) and tags (`t/`)
 * Support for marking a contact detail as 'private' (`pa/`) (`pe/`) (`pp/`)
 * View details of a person (`view` : shows non-private details), (`viewall` : shows all details)
+* Added a login feature before going into addressbook command line, password is "password"
```

Comment:
You may want to restructure this sentence. Presently it's more in the form of a commit message.